doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3042,6 +3045,15 @@ CIA-351: Connectivity issues with :ref:`lib_azure_iot_hub`
 Bluetooth libraries and services
 ================================
 
+.. _ncsdk_30288:
+
+.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0
+
+NCSDK-30288: The :c:func:`bt_hogp_rep_read` function of :ref:`hogp_readme` library does not forward ATT error code through the user callback
+  The library passes value of ``0`` to the user instead of the error code.
+
+  **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``0d227d82bbdf56a3c066021fa6c323a00107fe6f``).
```

Comment:
```suggestion
  **Workaround:** Manually cherry-pick and apply the commit ``0d227d82bbdf56a3c066021fa6c323a00107fe6f`` with the fix from the ``main`` branch.
```

Reply:
Similar approach was already used for most of the known issues. I wonder if we should change it here

Reply:
To be honest, I have no strong opinion which one is better. I think it just might be worth to keep it consistent among known issues page.

Reply:
I agree. Let's stick to the old convention.